assets/src/edit-story/elements/image/display.js

```diff
@@ -38,6 +39,27 @@ function ImageDisplay({ element, box }) {
   const { width, height } = box;
   const ref = useRef();
 
+  let initialSrcType = 'smallest';
+  let initialSrc =
+    resource.sizes?.['web-stories-thumbnail']?.source_url ||
+    resource.sizes?.medium?.source_url;
```

Comment:
Is `medium` the smallest reasonable size here?

Reply:
Yes, from what I found:

![image](https://user-images.githubusercontent.com/21036224/90406405-65c73080-e0a5-11ea-8405-0e32486eddd6.png)